lab-edward-smith/lib/readFiles.js

```diff
@@ -0,0 +1,15 @@
+const Promise  = require('promise');
```

Comment:
You don't need this as ES6 promises are supported in Node 4. So you can just use the without needing to require anything.